Makefile

```diff
@@ -29,4 +41,24 @@ test:
 	$(TEST)
 
 clean:
-	git checkout -- $(TARGET)
+	git checkout -- $(PHPBREW_PHAR)
+
+$(PHPBREW_PHAR): vendor \
+		$(PHAR_SRC_FILES) \
+		box.json.dist \
+		.git/HEAD
+	box compile
+	touch -c $@
+
+PHONY: vendor_install
+vendor_install:
```

Comment:
I added some extra targets to make sure the `vendor` is up to date.

In general one can use `composer install` as usual, but if you want to force it to make sure the Make targets are up to date then you can call `make vendor_install`, and it will update all the targets.